code/midtown/mmai/aiGoalFollowWayPts.cpp

```diff
@@ -19,3 +19,39 @@
 define_dummy_symbol(mmai_aiGoalFollowWayPts);
 
 #include "aiGoalFollowWayPts.h"
+
+#include "mmcar/car.h"
+#include "mmcityinfo/state.h"
+
+#include "aiData.h"
+#include "aiMap.h"
+
+b32 aiGoalFollowWayPts::Context()
+{
+    if (Car->Sim.MaxDamageScaled >= Car->Sim.CurrentDamage)
```

Comment:
Ok one more small thing, flip the comparison so it's easier to understand. `Car->Sim.CurrentDamage <= Car->Sim.MaxDamageScaled`